styles.css

```diff
@@ -104,6 +109,11 @@ a:hover::after
     }
 }
 
+#top-bar-brand-name
+{
+    cursor: pointer;
+}
+
 .search-bar .search-bar-input
 {
     outline: none;
```

Comment:
wrt class: .search-bar-input
Suggestion: font-family: inherit;
For the input, font-family can be inherited to make the font-style consistent throughout the page.

Reply:
Ooh right, I totally forgot about font-family of search bar, thanks!